content/en/tracing/services/service_page.md

```diff
@@ -70,14 +71,26 @@ The service page features summary cards with highlights on your service health.
 
 Our [Service Level Objectives (SLOs)][5] and [Incidents][6] summaries allow you to monitor the status of SLOs and ongoing incidents, so that you can keep performance goals top of mind. Click the cards to create a new SLO on the service or declare an incident. The [security signals][18] summary highlights how your services react to application threats.
 
+## Dependencies
+
+{{< callout url="https://docs.google.com/forms/d/1imGm-4SfOPjwAr6fwgMgQe88mp4Y-n_zV0K3DcNW4UA/edit" d_target="#signupModal" btn_hidden="true" btn_hidden="false" header="Opt in to the private beta!" >}}
+Inferred services and the new Service Page layout are in private beta. To request access, complete the form.
+{{< /callout >}}
+
+Use the dependency map to visualize service-to-service communication and gain insight into system components such as databases, queues, and third-party dependencies. You can group dependencies by type and filter by Requests, Latency, or Errors to identify slow or failing connections.
+
+For more information, see [New Service page and Inferred Services][20].
```

Comment:
Optional: Depending on whether you change the title for consistency, it should match here.
```suggestion
For more information, see [New Service page and inferred services][20].
```